### PR TITLE
adds links into head tag

### PR DESCRIPTION
**Pull request by cmizzi**

### CODE REVIEW

**Comment by greg0ire** [comment]
Please look for commit you can squash together. For instance, you can squash "adds changelog BC BREAK" with the commit that introduces this change. Both changes should not be separated.

**Comment by greg0ire** [comment]
Also, squash documentation commits with the things you document.

**Comment by greg0ire** [comment]
Looking nicer!

**Comment by cmizzi** [comment]
Thank you for all of your feedbacks :smile:

File: Seo/SeoPage.php [hunk 1]
```diff
@@ -34,11 +34,6 @@ class SeoPage implements SeoPageInterface
     /**
      * @var string
      */
-    protected $linkCanonical;
```
Comment (core23) [inline Seo/SeoPage.php]: Could you remove the BC break here? @cmizzi

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**